stdlib/messaging/rabbitmq/src/main/ballerina/rabbitmq/channel_listener.bal

```diff
@@ -14,20 +14,25 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/io;
 # Public Ballerina API - Ballerina RabbitMQ Message Listener.
 # To provide a listener to consume messages from RabbitMQ.
 #
 # + amqpChannel - Reference to a Ballerina RabbitMQ Channel.
-public type ChannelListener object {
+public type Listener object {
 
     *AbstractListener;
 
     private Channel? amqpChannel;
 
     # Initializes a Ballerina ChannelListener object with the given Connection object or connection parameters.
-    # Creates a Connection object if only the connection configuration is given.
+    # Creates a Connection object if only the connection configuration is given. Sets global QoS settings
+    # which will be applied to the entire channel.
     #
     # + connectionOrConnectionConfig - Holds a Ballerina RabbitMQ Connection object or the connection parameters.
+    # + prefetchCount - Maximum number of messages that the server will deliver, 0 if unlimited.
```

Comment:
```suggestion
    # + prefetchCount - Maximum number of messages that the server will deliver, 0 if unlimited.
```
```suggestion
    # + prefetchCount - Maximum number of messages that the server will deliver. It will be zero if unlimited.
```